IRescue/Core/DataTypes/RotationMatrix.cs

```diff
@@ -90,5 +95,34 @@ private static float[] CreateMatrixVector(float xRotation, float yRotation, floa
                 };
             return rot;
         }
+
```

Comment:
CreateMatrixVector(float xRotation, float yRotation, float zRotation)
Gets called in the constructor which accepts euler angles. That constructor does not get called in the current version of the unity code, but has been called a lot in the past and might be called again in the future.
CreateMatrixVector(Vector3 axis, float angle)
Does not get used anywhere anymore and can get removed indeed.